client/app/lib/constants/videoConstants.js

```diff
@@ -12,7 +12,7 @@ export const youtubeOpts = {
 
 export const videoDefaults = {
   volume: 0.8,
-  availablePlaybackRates: [0.5, 1, 1.5, 2, 2.5],
+  availablePlaybackRates: [0.5, 1, 1.25, 1.5, 1.75, 2, 2.25, 2.5],
```

Comment:
Shall we do the following instead `0.25, 0.5, 0.75, 1, 1.25, 1.5, 2.0, 2.5`. I think the 0.25 after 1.5x doesn't really make much of a difference, so no point including so many of those.

Reply:
Good idea, and it's closer to YouTube's playback rates too.